agentMET4FOF/metrological_agents.py

```diff
@@ -198,3 +201,43 @@ def custom_plot_function(self, data, sender_agent, **kwargs):
         else:
             trace = go.Scatter()
         return trace
+
+class MetrologicalGeneratorAgent(MetrologicalAgent):
+    """An agent streaming a sine signal
+
+    Takes samples from an instance of :py:class:`MetrologicalSineGenerator` with sampling frequency `sfreq` and
+    signal frequency `sine_freq` and pushes them sample by sample to connected agents via its output channel.
+    """
+
+    # The datatype of the stream will be MetrologicalSineGenerator.
```

Comment:
This comment probably should be dropped to not forget changing it if the implementation changes, as has happened now.

```suggestion
```